cmd/plasmacli/store/keystore.go

```diff
@@ -40,8 +40,8 @@ func InitKeystore(homeDir string) {
 	}
 }
 
-// Return iterator over accounts
-// returns db so db.close can be called
+// AccountIterator returns an iterator for accounts. It returns db so
+// db.close can be called.
```

Comment:
This should be a contract.
```suggestion
// CONTRACT: Caller is responsible for closing db after use.
```